pkg/destroy/aws/ec2helpers.go

```diff
@@ -778,36 +776,7 @@ func deleteEC2Volume(ctx context.Context, client *ec2v2.Client, id string, logge
 	return nil
 }
 
-func deleteEC2VPC(ctx context.Context, ec2Client *ec2v2.Client, elbClient *elb.ELB, elbv2Client *elbv2.ELBV2, id string, logger logrus.FieldLogger) error {
-	// first delete any Load Balancers under this VPC (not all of them are tagged)
-	v1lbError := deleteElasticLoadBalancerClassicByVPC(ctx, elbClient, id, logger)
-	v2lbError := deleteElasticLoadBalancerV2ByVPC(ctx, elbv2Client, id, logger)
-	if v1lbError != nil {
-		if v2lbError != nil {
-			logger.Info(v2lbError)
-		}
-		return v1lbError
-	} else if v2lbError != nil {
-		return v2lbError
-	}
-
-	for _, child := range []struct {
-		helper   func(ctx context.Context, client *ec2v2.Client, vpc string, failFast bool, logger logrus.FieldLogger) error
-		failFast bool
-	}{
-		{helper: deleteEC2NATGatewaysByVPC, failFast: true},      // not always tagged
-		{helper: deleteEC2NetworkInterfaceByVPC, failFast: true}, // not always tagged
-		{helper: deleteEC2RouteTablesByVPC, failFast: true},      // not always tagged
-		{helper: deleteEC2SecurityGroupsByVPC, failFast: false},  // not always tagged
```

Comment:
:thought_balloon: What about the default security group and the `main` route table? Those are not tagged with the cluster infraID and might block destroy, right?

Reply:
Ah, whoops! I just got the answer at the same time I asked :sweat_smile: 

The default SG [0] and main route table [1] cannot be deleted. They are cleaned up automatically when the VPC is deleted.

[0]  https://docs.aws.amazon.com/vpc/latest/userguide/default-security-group.html
[1] https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DeleteRouteTable.html

Reply:
And the existing destroy code also skip deleting them anyways...